tests/typowalking/wordwalker.js

```diff
@@ -5,7 +5,10 @@
 
 bender.editor = {
 	config: {
-		enterMode: CKEDITOR.ENTER_P
+		enterMode: CKEDITOR.ENTER_P,
+		nanospell: {
+			autostart: false
+		}
```

Comment:
otherwise, multiple walkers run at the same time, and can screw up the bookmark.